include/picongpu/plugins/openPMD/writer/ParticleAttribute.hpp

```diff
@@ -147,7 +147,7 @@ namespace picongpu
                 /* @todo check if always correct at this point,
                  * depends on attribute and MW-solver/pusher implementation
                  */
-                float_X const timeOffset = 0.0;
+                float_32 const timeOffset = 0.0;
                 record.setAttribute("timeOffset", timeOffset);
```

Comment:
offline discussed with @PrometheusPi please try to use 

```C++
record.setTimeOffset<float_X>(0.0);
```

For time offset there is a specific interface in OpenPMD maybe it is not allowed to set this with `setAttribute`

@franzpoeschel do you see why maybe a hard coded 32bit value could be nessesary?